<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>531</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>801</width>
    <height>531</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>801</width>
    <height>531</height>
   </size>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>English Class</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="signUpBtn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>425</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight SemiConde</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: none;
font: bold 20px;
padding: 0;
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(0, 0, 0, 0));
color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>400</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Not on English Class yet?</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>231</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ravie</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>true</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: none;</string>
    </property>
    <property name="title">
     <string>Sign In</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>62</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
      </font>
     </property>
     <property name="text">
      <string>UserAccount</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>112</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
      </font>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="account">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <stylestrategy>NoAntialias</stylestrategy>
      </font>
     </property>
     <property name="accessibleDescription">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);
border-style: solid;
border-width: 1px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pass">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <stylestrategy>NoAntialias</stylestrategy>
      </font>
     </property>
     <property name="accessibleDescription">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);
border-style: solid;
border-width: 1px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="signInBtn">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift SemiLight SemiConde</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-width: 1px;
border-radius: 5px;
font: bold 20px;
padding:2px;
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(0, 0, 0, 0));
color: rgb(0, 85, 255);</string>
     </property>
     <property name="text">
      <string>Sign In</string>
     </property>
    </widget>
    <widget class="QLabel" name="error">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>148</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>OCR A Extended</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string>Error</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>531</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
image: url(:/new/image/img/login.jpg);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/2785219.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>450</y>
      <width>600</width>
      <height>10</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QLabel" name="loading">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>400</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/loading.gif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>311</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/new/image/img/logormbg.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="admin">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/new/image/img/logormbg.png</normaloff>:/new/image/img/logormbg.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>signUpBtn</zorder>
   <zorder>label</zorder>
   <zorder>groupBox</zorder>
   <zorder>progressBar</zorder>
   <zorder>loading</zorder>
   <zorder>logo</zorder>
   <zorder>admin</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
